src/core/ext/filters/client_channel/lb_policy_factory.h

```diff
@@ -66,8 +66,9 @@ typedef struct grpc_lb_addresses {
 grpc_lb_addresses *grpc_lb_addresses_create(
     size_t num_addresses, const grpc_lb_user_data_vtable *user_data_vtable);
 
-/** Creates a copy of \a addresses. */
-grpc_lb_addresses *grpc_lb_addresses_copy(const grpc_lb_addresses *addresses);
+/** Creates a copy of \a addresses. When \a only_backends is true, does not 
```

Comment:
This comment now documents a parameter that no longer exists.

There should be no differences between this file and the version in master.

Reply:
I see. Mark, how do you spot such issues? I thought I am careful before this PR...